### PR TITLE
amixer: add page

### PR DESCRIPTION
**Pull request by telmotrooper**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/amixer.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# amixer
+
+> Mixer for ALSA soundcard driver.
+
+- Turn up master volume 10%:
+
+`amixer -D pulse sset Master 10%+`
```
Comment (pxgamer) [inline pages/linux/amixer.md]:
```suggestion
`amixer -D pulse sset Master {{10%+}}`
```

File: pages/linux/amixer.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Turn down master volume 10%:
+
+`amixer -D pulse sset Master 10%-`
```
Comment (pxgamer) [inline pages/linux/amixer.md]:
```suggestion
`amixer -D pulse sset Master {{10%-}}`
```

File: pages/linux/amixer.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# amixer
+
+> Mixer for ALSA soundcard driver.
+
+- Turn up master volume 10%:
+
+`amixer -D pulse sset Master 10%+`
+
+- Turn down master volume 10%:
```
Comment (pxgamer) [inline pages/linux/amixer.md]:
```suggestion
- Turn down the master volume by 10%:
```

File: pages/linux/amixer.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# amixer
+
+> Mixer for ALSA soundcard driver.
+
+- Turn up master volume 10%:
```
Comment (pxgamer) [inline pages/linux/amixer.md]:
```suggestion
- Turn up the master volume by 10%:
```

**Comment by telmotrooper** [comment]
@pxgamer Sure, just accepted your changes.

File: pages/linux/amixer.md (hunk 1 above)
Comment (pxgamer) [reply pages/linux/amixer.md]: Hi @telmotrooper, looks like this one was missed. Sorry. 👍

**Comment by telmotrooper** [comment]
I think I accepted an outdated suggestion, let me look give it a look when I get to work.

**Comment by telmotrooper** [comment]
@pxgamer Okay, made a new commit fixing the outdated suggestion I had accepted. It should be good now.

**Pull request merged**